src/SFA.DAS.EmployerIncentives.Api/Controllers/EmploymentCheckController.cs

```diff
@@ -6,14 +6,21 @@
 using SFA.DAS.EmployerIncentives.Enums;
 using System.Net;
 using System.Threading.Tasks;
+using SFA.DAS.EmployerIncentives.Domain.Interfaces;
 
 namespace SFA.DAS.EmployerIncentives.Api.Controllers
 {
     [Route("api/[controller]")]
     [ApiController]
     public class EmploymentCheckController : ApiCommandControllerBase
     {
-        public EmploymentCheckController(ICommandDispatcher commandDispatcher) : base(commandDispatcher) { }
+        private readonly ICollectionCalendarService _collectionCalendarService;
```

Comment:
I don't think this should be injected into the controller, maybe move the logic to the command handler?